static/js/components/CommentAttachmentPreview.jsx

```diff
@@ -124,10 +124,23 @@ const CommentAttachmentPreview = ({
   const theme = useTheme();
   const darkTheme = theme.palette.type === "dark";
 
+  function ressourceType(state) {
+    let type = "";
+    if (associatedResourceType === "gcn_event") {
+      type = state.gcnEvent.commentAttachment;
+    } else if (
+      associatedResourceType === "sources" ||
+      associatedResourceType === "spectra"
+    ) {
+      type = state.source.commentAttachment;
+    } else {
+      console.log(`Nothing corresponds to a right ${associatedResourceType}`);
```

Comment:
We probably dont need this else, right? Otherwise I think this should be a showNotification error, because the log here won't help.

Reply:
True, I don't think we need to keep it